e2e/create-cli-e2e/project.json

```diff
@@ -18,6 +18,6 @@
       }
     }
   },
-  "implicitDependencies": ["create-cli"],
+  "implicitDependencies": ["models", "utils", "core", "cli", "create-cli"],
```

Comment:
Shouldn't it be enough to set `cli` as implicit dependency? The `core`, `models` and `utils` dependencies would then automatically become indirect dependencies based on Nx graph.

Same goes for other `e2e/*-e2e/project.json`s.

Reply:
Related to https://github.com/code-pushup/cli/pull/810#discussion_r1738527881

Reply:
ATM we don't leverage the graph for that. The reason is I did not manage to pass the args to dependencies as well as some edge case for the existing CLI-e2e tests. 

Have it on my roadmap!

Reply:
Updated PR, managed finally to forward target params 🎉